challenge_3/python/bryantpq/find_majority.py

```diff
@@ -0,0 +1,17 @@
+def find_majority(a):
+    d = {}
+    l = len(a)
+    for i in a:
+        if i in d:
+            d[i] += 1
+            if d[i] >= l / 2:
```

Comment:
If `d[i] = 10` and `l = 20` is `i` the majority element? Remember that "The majority element is the element that appears **more** than ⌊ n/2 ⌋ times."